src/bluesky/callbacks/olog.py

```diff
@@ -173,5 +173,5 @@ def start(self, doc):
             pformat(doc),
         )
         olog_status = self.client.log(document_content, logbooks=self.logbook)
-        logger.debug("client.log returned %s" % olog_status)
+        logger.debug(f"client.log returned {olog_status}")
```

Comment:
```suggestion
        logger.debug(f"client.log returned %s", olog_status)
```

It is best-practice to defer stringy-ifying things passed to loggers to avoid paying the cost if there is nothing consuming the log.

Reply:
Isn't `logger` the exception to use of f strings? The point of *no f strings in logger methods* is that the method chooses whether to evaluate the arguments based on the log level. If an f string is used, the arguments are always evaluated, which defeats one of the features of the `logger`.

Reply:
```suggestion
        logger.debug("client.log returned %s", olog_status)
```

Reply:
Deleted my suggestion.  When first observed this, my phone showed it differently.  My comments and suggestion restated @tacaswell 's point.

Reply:
Ah then, we may have to change the ruff linter config maybe? The current version of ruff `0.4.2` does not like percent strings and fails, `0.4.1` passes. Any suggestion @tacaswell?

Reply:
Or I guess add `# noqa` to these lines

Reply:
One more question, should I leave other f string conversions as is? i.e. print statements or stick a noqa there as well?

Reply:
Great then I think we are all set. Checks and tests passed!